static/js/fx-bento.js

```diff
@@ -6,6 +6,7 @@ function getFxAppLinkInfo(localizedBentoStrings, referringSiteURL) {
     [localizedBentoStrings.fxPocket, "https://app.adjust.com/hr2n0yz?engagement_type=fallback_click&fallback=https%3A%2F%2Fgetpocket.com%2Ffirefox_learnmore%3Fsrc%3Dff_bento&fallback_lp=https%3A%2F%2Fapps.apple.com%2Fapp%2Fpocket-save-read-grow%2Fid309601447", "pocket"],
     [localizedBentoStrings.fxDesktop, `https://www.mozilla.org/firefox/new/?utm_source=${referringSiteURL}&utm_medium=referral&utm_campaign=bento&utm_content=desktop`, "fx-desktop"],
     [localizedBentoStrings.fxMobile, `http://mozilla.org/firefox/mobile?utm_source=${referringSiteURL}&utm_medium=referral&utm_campaign=bento&utm_content=desktop`, "fx-mobile"],
+    [localizedBentoStrings.fxVpn, "https://www.mozilla.org/products/vpn/", "fx-vpn"],
```

Comment:
Resubmitting this request again as it is still outstanding. We need to add UTM params: 

```suggestion
    [localizedBentoStrings.fxVpn, "https://www.mozilla.org/products/vpn?utm_source=${referringSiteURL}&utm_medium=referral&utm_campaign=bento&utm_content=desktop/", "fx-vpn"],
```

Reply:
Should I add one for monitor as well? It's currently only `https://monitor.firefox.com/` on L5

Reply:
Added UTM params for monitor and VPN.